lib/text.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+class Text
+  def initialize
+    account_sid = 'ACa353025d1bc42a06b259ff9332279f8d'
+    auth_token = '30e174a43e758ef20e98c283551427e0'
```

Comment:
I think it'd be better to use env variables for these too, as they could let someone get into your account